app/src/pages/prompt/promptLoader.tsx

```diff
@@ -14,6 +14,9 @@ import { promptLoaderQuery } from "./__generated__/promptLoaderQuery.graphql";
  */
 export async function promptLoader(args: LoaderFunctionArgs) {
   const { promptId } = args.params;
+  // @TODO: fragments fetch _all_ prompt versions, without pagination.
```

Comment:
You mean that it's fetching all because there is a fragment that's unbound in pagination args?

Reply:
Yeah. Every fragment that fetches prompt versions just gets all of them because different parts of the page need different amounts of prompt versions and you can't mix and match pagination